cfgov/unprocessed/css/molecules/global-banner.less

```diff
@@ -38,7 +38,9 @@
     }
 
     .o-expandable_target {
-        .btn__secondary();
+        // Taken from .a-btn__secondary.
+        background-color: @btn__secondary-bg;
```

Comment:
We would have to refactor the code in CF to not use the [nested syntax](https://github.com/cfpb/capital-framework/blob/canary/src/cf-buttons/src/atoms/buttons.less#L65). In this case since it's specifically styling for the global banner, which is only used on build anyway, and the styling added for `a-btn__secondary` is minimal, I think including the styles directly makes sense vs touching CF.